Sources/FuzzilliCli/main.swift

```diff
@@ -135,6 +137,7 @@ let storagePath = args["--storagePath"]
 var resume = args.has("--resume")
 let overwrite = args.has("--overwrite")
 let exportStatistics = args.has("--exportStatistics")
+let statisticsExportInterval = args.double(for: "--statisticsExportInterval") ?? 10.0
```

Comment:
Can we just make it a UInt instead? If there's need for sub-minute intervals, we should probably just specify it in seconds, but I think minutes is good enough.